stratum/hal/lib/barefoot/bf.proto

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020-present Open Networking Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+option cc_generic_services = false;
+
+package stratum.hal.barefoot;
+
+import "p4/config/v1/p4info.proto";
+
+// This message describes the public Barefoot specific definition of a single P4
+// program. It is passed to Stratum as the p4 device config of the P4Runtime
+// ForwardingPipelineConfig message.
+message BfPipelineConfig {
+  message Profile {
```

Comment:
Would be good to add comments about the expected values for each field. This is obvious by looking at the implementation of the pipeline builder tools so I don't think there's a sensitivity issue.

Reply:
ok. I'll add my own to avoid any legal hazards.